node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala

```diff
@@ -174,6 +184,31 @@ class UtilsRouteSpec extends RouteSpec("/utils") with RestAPISettingsHelper with
 
+  val dApp =
```

Comment:
please add Verifier function to this script

Reply:
+